src/Microsoft.Xaml.Behaviors.DesignTools/MetadataTableProvider.cs

```diff
@@ -12,16 +13,16 @@ namespace Microsoft.Xaml.Behaviors.DesignTools
 {
     internal class MetadataTableProvider : IProvideAttributeTable
     {
-        private AttributeTableBuilder _attributeTableBuilder;
+        private AttributeTableBuilder attributeTableBuilder;
 
         // Accessed by the designer to register any design-time metadata.
         public AttributeTable AttributeTable
         {
             get
             {
-                if (_attributeTableBuilder == null)
+                if (this.attributeTableBuilder == null)
```

Comment:
In this repo we have different conventions: no this.

Reply:
Removed.